RNSentry.podspec

```diff
@@ -10,15 +13,30 @@ Pod::Spec.new do |s|
   s.homepage       = "https://github.com/getsentry/sentry-react-native"
   s.source         = { :git => 'https://github.com/getsentry/sentry-react-native.git', :tag => "#{s.version}"}
 
-  s.ios.deployment_target = "8.0"
+  s.ios.deployment_target = "11.0"
   s.osx.deployment_target = "10.10"
-  s.tvos.deployment_target = "9.0"
+  s.tvos.deployment_target = "11.0"
```

Comment:
Does this affect the min. version of iOS for compatibility?

Reply:
Yes,
current cocoa-sdk support 9.0 but react-native 0.70 supports 12.4.

Reply:
so I should be 12 then? since we support RN >= 0.70

Reply:
Yea, I took this from the example lib implementation, but I guess it was implemented before 0.70 release.

I'll match it with RN.